x-pack/plugins/data_enhanced/server/search/session/session_service.ts

```diff
@@ -229,40 +255,69 @@ export class SearchSessionService
         touched: new Date().toISOString(),
         idMapping: {},
         persisted: false,
+        realmType,
+        realmName,
+        username,
         ...attributes,
```

Comment:
It appears this is the case, but to confirm: it's not possible for `attributes` to accidentally override any of `realmType`, `realmName`, or `username`, right?